Domain-Property missing EIP section of template-two (part two)?

Running CloudFormation with template-two.json gives me this error:

`CREATE_FAILED AWS::EC2::EIP EIP You must specify an allocation id when mapping an address to a VPC instance`

From reading the [documentation for AWS::EC2::EIP](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-eip.html) I feel like we need a `Domain`-property for the EIP. 

Configuring it like this worked for me: 

```
"EIP" : {
  "Type" : "AWS::EC2::EIP",
  "DependsOn" : "VPCGatewayAttachment",
  "Properties" : {
    "InstanceId" : { "Ref" : "EC2Instance" },
    "Domain": "vpc"
  }
}
```

--- a/part-four/infrastructure/provisioning/infrastructure-template.json
+++ b/part-four/infrastructure/provisioning/infrastructure-template.json
@@ -176,7 +176,7 @@
       "Type" : "AWS::EC2::EIP",
       "Properties" : {
         "InstanceId" : { "Ref" : "EC2InstanceCIMaster" },
-        "Domain" : "VPC"
+        "Domain" : "vpc"
       },
       "DependsOn" : "VPCGatewayAttachment"
     },
@@ -185,7 +185,7 @@
       "Type" : "AWS::EC2::EIP",
       "Properties" : {
         "InstanceId" : { "Ref" : "EC2InstanceCISlave" },
-        "Domain" : "VPC"
+        "Domain" : "vpc"
       },
       "DependsOn" : "VPCGatewayAttachment"
     },

--- a/part-three/infrastructure/provisioning/infrastructure-template.json
+++ b/part-three/infrastructure/provisioning/infrastructure-template.json
@@ -162,7 +162,7 @@
       "Type" : "AWS::EC2::EIP",
       "Properties" : {
         "InstanceId" : { "Ref" : "EC2InstanceCIMaster" },
-        "Domain" : "VPC"
+        "Domain" : "vpc"
       },
       "DependsOn" : "VPCGatewayAttachment"
     },
@@ -171,7 +171,7 @@
       "Type" : "AWS::EC2::EIP",
       "Properties" : {
         "InstanceId" : { "Ref" : "EC2InstanceCISlave" },
-        "Domain" : "VPC"
+        "Domain" : "vpc"
       },
       "DependsOn" : "VPCGatewayAttachment"
     },

--- a/part-two/templates/template-two.json
+++ b/part-two/templates/template-two.json
@@ -125,7 +125,7 @@
       "Type" : "AWS::EC2::EIP",
       "Properties" : {
         "InstanceId" : { "Ref" : "EC2Instance" },
-        "Domain" : "VPC"
+        "Domain" : "vpc"
       },
       "DependsOn" : "VPCGatewayAttachment"
     },

